doc/source/building/index.rst

```diff
@@ -376,6 +376,37 @@ interface is self-documenting, so please see ``python dev.py --help`` and
 ``python dev.py <subcommand> --help`` for detailed guidance.
 
 
+.. admonition:: IDE support & editable installs
+
+    While the ``dev.py`` interface is our recommended way of working on SciPy,
+    it has one limitation: because of the custom install location, setting
+    breakpoints inside an IDE or debugger does not work as expected. This will
```

Comment:
Nit: strictly speaking, `pdb` breakpoints work just fine: drop a `breakpoint()` into the source, run `python dev.py test -t something/to/to/trigger/this/code/path` and you drop into pdb.
So it's some IDEs' problem?

Reply:
Yes, the problem is that in an IDE, you do something like clicking on the sidebar to set a breakpoint, and then the IDE knows how to tell its debugger that it's line X in file Y. Only works if you don't copy file Y ....

For CLI debuggers it's the same - works if you save the breakpoint in the file, not if you set it external to the file somehow.